pkgs/development/lua-modules/overrides.nix

```diff
@@ -546,7 +546,6 @@ in
 
   neotest  = prev.neotest.overrideAttrs(oa: {
     # A few tests fail for strange reasons on darwin
-    doCheck = !stdenv.isDarwin;
```

Comment:
it should probably be
```diff
-    # A few tests fail for strange reasons on darwin
-    doCheck = !stdenv.isDarwin;
+    doCheck = true;
```
because https://discourse.nixos.org/t/questions-about-docheck-and-checkinputs/21827